<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>board_mini</class>
 <widget class="QWidget" name="board_mini">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>78</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>78</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#mini_frame{
	border-radius: 10px;
	background-color: #1f1f23;
}

QFrame#mini_frame:hover{
	background-color: #333333;
}

QLabel{
	color: white;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="mini_frame">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_tarjeta">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/board/images/board_small.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_nombre">
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_tags" native="true">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 9pt &quot;Inter&quot;;
	padding: 3px;
	color: white;
	background-color: #666;
	border-radius: 5px;
}

QLabel:hover{
	background-color: #555;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_sensores" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
